### ISSUE TITLE
Headings are not structured - Facilities named buildings

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9313]
# Motivation

SiteImprove is reporting the headings are not structured issue on Facilities named buildings. This is an accessibility best practice conformance issue.

https://my2.siteimprove.com/Inspector/1005109/A11Y/Page?pageId=65062348353&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r53/failed/UCtqaldTRVY=;YWMwNzM0ZWI=

# Additional Info

It appears we are outputting the honorees name as an h4. The previous heading is an h1.
